elements/rh-navigation-secondary/rh-navigation-secondary.ts

```diff
@@ -142,9 +142,14 @@ export class RhNavigationSecondary extends LitElement {
     this.addEventListener('overlay-change', this.#onOverlayChange);
     this.addEventListener('focusout', this.#onFocusout);
     this.addEventListener('keydown', this.#onKeydown);
+    window.addEventListener('keyup', this.#onKeyup.bind(this));
```

Comment:
this will add a new event listener to `globalThis` for every instance of the element.

Practically, that might not be a huge deal since this is a nav bar, nonetheless, we should prefer to add this listener statically.